src/containers/deactivate-account-page/store/sagas.ts

```diff
@@ -0,0 +1,91 @@
+import { waitFor } from '@testing-library/dom'
+import { delay } from 'redux-saga'
+import { call, fork, put, select, takeEvery } from 'redux-saga/effects'
+
+import { ID } from 'common/models/Identifiers'
+import { getAccountUser, getUserId } from 'common/store/account/selectors'
+import { Nullable } from 'common/utils/typeUtils'
+import AudiusBackend from 'services/AudiusBackend'
+import { waitForBackendSetup } from 'store/backend/sagas'
+import { requestConfirmation } from 'store/confirmer/actions'
+import { confirmTransaction } from 'store/confirmer/sagas'
+import {
+  getConfirmCalls,
+  getConfirmLength,
+  getIsConfirming
+} from 'store/confirmer/selectors'
+import { waitForValue } from 'utils/sagaHelpers'
+import { signOut } from 'utils/signOut'
+
+import {
+  afterDeactivationSignOut,
+  deactivateAccount,
+  deactivateAccountFailed
+} from './slice'
+
+const DEACTIVATE_CONFIRMATION_UID = 'DEACTIVATE'
+
+function* handleDeactivateAccount() {
+  try {
+    yield call(waitForBackendSetup)
+    const accountUserId: Nullable<ID> = yield select(getUserId)
+    const userMetadata: ReturnType<typeof getAccountUser> = yield select(
+      getAccountUser
+    )
+    yield put(
+      requestConfirmation(
+        DEACTIVATE_CONFIRMATION_UID,
+        function* () {
+          const { blockHash, blockNumber } = yield call(
+            AudiusBackend.updateCreator,
+            { ...userMetadata, is_deactivated: true },
+            accountUserId /* note: as of writing, unused parameter */
```

Comment:
odd.

Reply:
Yeah we seem to pull `user_id` from the `userMetadata` instead